articles/machine-learning/how-to-prepare-datasets-for-automl-images.md

```diff
@@ -76,10 +80,44 @@ az ml data create -f [PATH_TO_YML_FILE] --workspace-name [YOUR_AZURE_WORKSPACE]
 [!Notebook-python[] (~/azureml-examples-main/sdk/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items.ipynb?name=upload-data)]
 ---
 
+If you already have your data present in an existing datastore and want to create data asset out of it, you can do so by providing path to the data in datastore as shown below, instead of providing path on your local machine.
```

Comment:
Do we need to do this if data is already in datastore ? Would using that path directly in the jsonl file not work ?

Reply:
here's my understanding/observation: I directly use path to datastore directly in jsonl, then it creates a new-different-data assets at runtime, for every single run. So, for n runs it'll create n different data assets, creating redundancy and messiness in data asset catalog for user. Hence, there is some benefit I see in creating data asset, even when datastore path can be directly used......What do u think ?

Reply:
Ohh I see. It makes sense to have this section then. Thanks for the explanation